torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -433,7 +433,7 @@ export default function Hud() {
   return (
     <>
       <Head>
-        <title>PyTorch CI HUD {title}</title>
+        <title>CI HUD {title}</title>
```

Comment:
This was needed! Could even do:

```suggestion
        <title>HUD {title}</title>
```